server/plugin/command.go

```diff
@@ -85,6 +88,99 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) []string {
+	mutedUsernameBytes, err := p.API.KVGet(userInfo.UserID + "-muted-users")
+	if err != nil {
+		return nil
+	}
+	mutedUsernames := string(mutedUsernameBytes)
+	var mutedUsers []string
+	if len(mutedUsernames) == 0 {
+		return mutedUsers
+	}
+	mutedUsers = strings.Split(mutedUsernames, ",")
+	return mutedUsers
+}
+
+func (p *Plugin) handleMuteList(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	return "Your muted usernames: " + strings.Join(mutedUsernames, ", ")
+}
+
+func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	var mutedUsers string
+	if len(mutedUsernames) > 0 {
+		// , is a character not allowed in github usernames so we can split on them
+		mutedUsers = strings.Join(mutedUsernames, ",") + "," + username
+	} else {
+		mutedUsers = username
+	}
+	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte(mutedUsers)); err != nil {
+		return "Error occurred saving list of muted users"
+	}
+	return "Your muted usernames: " + mutedUsers
```

Comment:
1/5 it's a bit confusing that the whole list is returned instead of only the user that was newly added. Maybe something like `$USER is now muted for you`